gems/aws-sdk-core/lib/aws-sdk-core/binary/event_parser.rb

```diff
@@ -53,8 +58,10 @@ def parse_event(raw_event)
         event_type = raw_event.headers.delete(":event-type").value
         # content_type = raw_event.headers.delete(":content-type").value
 
-        # TODO Pending
-        # if event_type == 'initial-response' end
+        # Pending
+        raise Aws::Errors::EventStreamParserError.new(
```

Comment:
Style point: Don't use inline `if` when the code is not a single line. Prefer a wrapping `if` statement.